src/appshell/view/preferences/advancedpreferencesmodel.cpp

```diff
@@ -96,28 +96,46 @@ void AdvancedPreferencesModel::load()
     for (auto it = items.cbegin(); it != items.cend(); ++it) {
         if (it->second.canBeManuallyEdited) {
             m_items << it->second;
+
+            muse::Settings::Key key = it->second.key;
+            settings()->valueChanged(key).onReceive(this, [this, key](const Val& val) {
```

Comment:
I believe each subsequent call to `settings()->valueChanged(key).onReceive(this` (with emphasis on the latter `this`) either overrides the previously registered callback or does nothing

Reply:
Can you elaborate? Each subsequent call is for a different setting (key) => different notification channel.

Reply:
Ah sorry, you're definitely right!